api/Notification.json

```diff
@@ -13,10 +13,11 @@
             ]
           },
           "chrome_android": {
-            "version_added": "25",
+            "version_added": "42",
+            "partial_implementation": true,
             "notes": [
-              "Starting in Chrome 49, notifications do not work in incognito mode.",
-              "Before Chrome 42, service worker additions were not supported."
+              "Notifications in Chrome for Android are only available through service workers. Developers must use <a href='https://developer.mozilla.org/docs/Web/API/ServiceWorkerRegistration/showNotification'><code>ServiceWorkerRegistration.showNotification()</code></a> instead.",
+              "Starting in Chrome 49, notifications do not work in incognito mode."
```

Comment:
Any reason that this note isn't copied to other browsers? And is there a better place to put it, directly on the API that doesn't work?